### ISSUE TITLE
Beta8 release

### ISSUE THREAD

**Issue by peterblazejewicz** [OmniSharp/generator-aspnet#386]
Just tasks:
- [x] branch #387 
- [x] label for release
- [x] Class Library #381
- [x] Console Application #380 
- [x] Empty Application #382 
- [x] Nancy ASP.NET Application #388 
- [x] Web API Application #383
- [x] Web Application #385 
- [x] Web Application Basic [without Membership and Authorization] #384 
- [x] Dockerfile subgenerator update #395 
- [x] development note (feed, api, dnvm, etc) #390 
- [x] review issues labelled with `beta8` label against update
- [x] documentation review

The ASP.NET roadmap states `beta8` release for 12th:
https://github.com/aspnet/Home/wiki/Roadmap

- ~~Unit Test project #389~~
The #389 moved out of the relase. See #389 comments why

**Comment by peterblazejewicz** [comment]
@spboyer 
Should we rebase `beta8` with changes from `master` after #366?
I've checked locally and there seems to be no conflicting changes between 'master` and `beta8` - at least when rebasing my local PRs

**Comment by peterblazejewicz** [comment]
animation update
![yo](https://cloud.githubusercontent.com/assets/14539/10418056/b72ae284-7050-11e5-99db-5a0cda8f0ac1.gif)

**Comment by peterblazejewicz** [comment]
Unit Test postponed after officially supported by related project:
https://github.com/OmniSharp/generator-aspnet/issues/389

**Comment by peterblazejewicz** [comment]
@sayedihashimi 
reviewed task list for beta8 and closing this
Thanks!

**Comment by spboyer** [comment]
@sayedihashimi @peterblazejewicz - dnvm beta8 is out, can we release?  Al items are in correct?

**Comment by sayedihashimi** [comment]
Its going out soon, I'm waiting to hear back and I will release it.

**Comment by peterblazejewicz** [comment]
all issues/prs are closed. The last remaning topic are UnitTests:
https://github.com/OmniSharp/generator-aspnet/issues/389
that will be upgraded later (as agreed)

**Comment by peterblazejewicz** [comment]
@sayedihashimi 
MS is releasing samples wit both `web` and `kestrel` in `project.json` they are using different notation for ports:
https://github.com/aspnet/Home/blob/dev/samples/1.0.0-beta8/HelloWeb/project.json#L21
https://github.com/aspnet/Home/blob/dev/samples/1.0.0-beta8/HelloMvc/project.json#L20-L21

We are using just `web` as provided in `beta8` samples.
The same difference in Docker file:
https://github.com/aspnet/Home/blob/dev/samples/1.0.0-beta8/HelloWeb/Dockerfile

cc @spboyer

**Comment by spboyer** [comment]
Ugh

**Comment by peterblazejewicz** [comment]
the `web.config` thing is strictly VS addition:
> You’ll notice that the ASP.NET 5 project templates in Visual Studio have been updated to include the following web.config file ....

http://blogs.msdn.com/b/webdev/archive/2015/10/15/announcing-availability-of-asp-net-5-beta8.aspx
We should discuss this file later after beta8, it adds noice

**Comment by sayedihashimi** [comment]
OK I'm going to release. I noticed there are changes in master that are not in beta8 and vice versa. So I'm thinking to merge from beta8->release and publish that. After that we can merge from release/beta8->master and then re-release w/ xunit and namespace changes.

@spboyer @peterblazejewicz please let me know if you have any concerns.

**Comment by spboyer** [comment]
We should be able to go beta8->master->release 

I'd like to get namespaces in the release for sure. The diff between master and 8 should be no issue

**Comment by sayedihashimi** [comment]
@spboyer ok sounds good. I'll give @peterblazejewicz a few minutes to reply but he may be sleeping.

**Comment by peterblazejewicz** [comment]
I've got a clean rebase of master on beta8, without conflicts:
```
generator-aspnet git:(beta8) git rebase master
```
![image](https://cloud.githubusercontent.com/assets/14539/10526398/e48c56d0-738a-11e5-8cc6-ad0f5f8a2493.png)

It means that we could do either way I think. I'd do rebase to rewrite history as we are merging changes of beta8 to master and then to release.

**Comment by sayedihashimi** [comment]
@peterblazejewicz ok can you push the changes since you already have it ready?

**Comment by spboyer** [comment]
@peterblazejewicz love his `rebase`. Is that why you have 200k commits?

**Comment by peterblazejewicz** [comment]
all green here:
https://github.com/OmniSharp/generator-aspnet/pull/421

**Comment by peterblazejewicz** [comment]
> Is that why you have 200k commits?

:joy_cat:

**Comment by sayedihashimi** [comment]
Thanks everyone, the release build has started https://travis-ci.org/OmniSharp/generator-aspnet/builds/85611394

**Comment by peterblazejewicz** [comment]
So closing, we have to starts issues with web/kestrel great debate (+ web.config) elsewhere :D

**Comment by spboyer** [comment]
@peterblazejewicz did you open a new issue on the "web/kestrel great debate (+ web.config)" ...

**Comment by peterblazejewicz** [comment]
@spboyer 
No, not yet. 
Let me to start two tickets for above here
